### PR TITLE
Fix links to examples

### PR DESCRIPTION
**Pull request by alexmojaki**

### CODE REVIEW

**Comment by alexmojaki** [comment]
@googlebot I signed it!

**Review by surma — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Comlink turns this messaged-based API into a something more developer-friendly b
 
 ## Examples
 
-### [Running a simple function](https://github.com/GoogleChromeLabs/comlink/tree/master/docs/examples/01-simple-example)
+### [Running a simple function](./docs/examples/01-simple-example)
```
Comment (surma) [inline README.md:34]: This won’t work if you look at the README from the repo’s landing page (`https://github.com/GoogleChromeLabs/comlink`). I think full URLs work best here

**Review by surma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alexmojaki) [reply README.md:34]: I was copying the existing link further down which seems to work fine:

```md
### [`SharedWorker`](./docs/examples/07-sharedworker-example)
```

Comment (surma) [reply README.md:34]: I didn’t catch that when I reviewed that, but it magically works. Amazing. never mind me then

**Review by surma — APPROVED** [review]

**Comment by surma** [comment]
Cheers!

**Pull request merged**